draft-ietf-dnsop-svcb-https.md

```diff
@@ -895,7 +902,8 @@ mandatory", i.e. mandatory if they are present in an RR.  The SvcParamKey
 any automatically mandatory keys that are present.
 
 A ServiceMode RR is considered "compatible" with a client if the client
-implements support for all its mandatory keys.  If the SVCB RRSet contains
+recognizes all the mandatory keys, and their values indicate that successful
```

Comment:
Should we rename this subsection from "mandatory" to "ServiceMode RR compatibility and mandatory keys" ?  That might be more descriptive?

Reply:
Done